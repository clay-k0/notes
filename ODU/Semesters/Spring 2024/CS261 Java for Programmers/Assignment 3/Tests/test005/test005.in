4
A 30 
B 75 
C 30 
D 90 
F 10 
E 10
