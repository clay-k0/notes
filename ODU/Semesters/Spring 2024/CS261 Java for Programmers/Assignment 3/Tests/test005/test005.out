225:  A:30  B:75  C:30  D:90
